AutoMapper.OData.EFCore.Tests/GetQueryTests.cs

```diff
@@ -220,6 +220,36 @@ void Test(ICollection<OpsTenant> collection)
             }
         }
 
+        [Fact]
+        public void BuildingsFilterNameDisableConstantParameterization()
+        {
+            string query = "/corebuilding?$filter=contains(Name, 'Two L2')";
+            Test(GetQuery<CoreBuilding, TBuilding>(query, querySettings: new() { ODataSettings = new() { EnableConstantParameterization = false } }));
+
+            void Test(IQueryable<CoreBuilding> queryable)
+            {
+                string sqlQuery = queryable.ToQueryString();
+                Assert.Contains("LIKE N'%Two L2%'", queryable.ToQueryString());           
+                Assert.DoesNotContain("DECLARE", sqlQuery);
+                Assert.DoesNotContain("ESCAPE", sqlQuery);
+            }
+        }
+
+        [Fact]
+        public void BuildingsFilterNameEnableConstantParameterization()
+        {
+            string query = "/corebuilding?$filter=contains(Name, 'Two L2')";
+            Test(GetQuery<CoreBuilding, TBuilding>(query, querySettings: new() { ODataSettings = new() { EnableConstantParameterization = true } }));
+
+            void Test(IQueryable<CoreBuilding> queryable)
+            {
+                string sqlQuery = queryable.ToQueryString();
```

Comment:
I'm not sure if it's the best way of testing my changes. Suggestions are welcome.